src/ServerConfig.cc

```diff
@@ -89,6 +89,7 @@ class ignition::gazebo::ServerConfig::PluginInfoPrivate
 
   /// \brief XML elements associated with this plugin
   public: sdf::ElementPtr sdf = nullptr;
+
```

Comment:
```suggestion
```

Reply:
Done in [afd30fb](https://github.com/gazebosim/gz-sim/pull/1352/commits/afd30fbb79c7c81cf9d7bb056179fa6cd92a0d94)